docs/conf.py

```diff
@@ -0,0 +1,353 @@
+# -*- coding: utf-8 -*-
+#
+# ReSolve documentation build configuration file, created by
+# sphinx-quickstart on Thu Nov 2 13:13:18 2023.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sys
+import os
+import shlex
+
+# Call doxygen in ReadtheDocs
+# read_the_docs_build = os.environ.get('READTHEDOCS', None) == 'True'
+# if read_the_docs_build:
+
+# Modify Doxyfile for ReadTheDocs compatibility
+with open('./doxygen/Doxyfile.in', 'r') as f:
+    fdata = f.read()
+fdata = fdata.replace('@PROJECT_SOURCE_DIR@', '.')
+with open('./doxygen/Doxyfile.in', 'w') as f:
+    f.write(fdata)
+    
+# Call doxygen
+from subprocess import call
+call(['doxygen', "./doxygen/Doxyfile.in"])
+
+
+# Get current directory
+conf_directory = os.path.dirname(os.path.realpath(__file__))
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#sys.path.insert(0, os.path.abspath('.'))
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.graphviz',
+    'sphinx.ext.autodoc',
+    'sphinx.ext.doctest',
+    'sphinxcontrib.jquery',
+    'sphinx.ext.todo',
+    'sphinx.ext.coverage',
+    'sphinx.ext.mathjax'
+]
+
+# The main toctree document.
+master_doc = 'index'
+
+# Add any paths that contain templates here, relative to this directory.
+# I think this is right:
+templates_path = [os.path.join(conf_directory, 'sphinx/_templates')]
+# Other variations:
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+# source_suffix = ['.rst', '.md']
+source_suffix = ['.rst', '.html']
+
+# The encoding of source files.
+#source_encoding = 'utf-8-sig'
+
+project = 'ReSolve'
+copyright = '2023, Kasia Świrydowicz, Slaven Peles'
+author = 'Kasia Świrydowicz, Slaven Peles'
```

Comment:
Copyright belongs to UT Battelle and Battelle Memorial Institute.
```suggestion
copyright = '2023, UT-Battelle, LLC, and Battelle Memorial Institute'
author = 'Kasia Świrydowicz, Slaven Peles'
```